### PR TITLE
add variables

### PR DESCRIPTION
**Pull request by AlexIaInder**

### CODE REVIEW

File: src/sass/utils/_variables.scss [hunk 1]
```diff
@@ -1 +1,19 @@
-// ЗАГАЛЬНІ ЗМІННІ 
\ No newline at end of file
+// ЗАГАЛЬНІ ЗМІННІ
+:root {
+  // Fonts
+  --main-font: 'Caveat Brush', sans-serif;
```
Comment (dankozz1t) [inline src/sass/utils/_variables.scss:4]: This font is cursive family

font-family: 'Caveat Brush', cursive;

[the remark this answers is not in the document]
Comment (AlexIaInder) [reply src/sass/utils/_variables.scss]: fix

File: src/sass/utils/_variables.scss [hunk 2]
```diff
@@ -1 +1,19 @@
...
+  --secondary-font: 'Montserrat', sans-serif;
+
+  // Text colors
+  --main-text-cl: #2e2520;
+  --button-text-cl: #ffffff;
+  --brown-text-cl: #ab816c;
+  --placeholder-text-cl: rgba(171, 129, 108, 0.5);
+  --donut-title-text-cl: #000000;
+  --hover-focus-text-cl: #876655;
```
[the remark this answers is not in the document]
Comment (AlexIaInder) [reply src/sass/utils/_variables.scss]: fix

File: src/sass/utils/_variables.scss [hunk 3]
```diff
@@ -1 +1,19 @@
...
+// ЗАГАЛЬНІ ЗМІННІ
+:root {
+  // Fonts
+  --main-font: 'Caveat Brush', sans-serif;
+  --secondary-font: 'Montserrat', sans-serif;
+
+  // Text colors
+  --main-text-cl: #2e2520;
+  --button-text-cl: #ffffff;
+  --brown-text-cl: #ab816c;
```
[the remark this answers is not in the document]
Comment (AlexIaInder) [reply src/sass/utils/_variables.scss]: fix

**Review by dankozz1t — APPROVED** [review]
Good job!

**Pull request merged**